nixos/modules/config/malloc.nix

```diff
@@ -22,6 +22,15 @@ let
       '';
     };
 
+    jemalloc-mariadb = {
```

Comment:
One last question for clarification. Does it crash because of the option we enable or because of the incompatible version?

Reply:
Rechecked.
There was a conflict with this parameter.
```
  systemd.services.mysql = with pkgs; {
    environment = {
      LD_PRELOAD = "${pkgs.jemalloc450}/lib/libjemalloc.so";
    };
```
Removed.
Now with this parameter:
```
  environment.memoryAllocator.provider = "graphene-hardened";
```
MySQL service crashed.
With "jemalloc" and "scudo" correct work: